### ISSUE TITLE
Unable to interact with game on https://athenacrisis.com

### ISSUE THREAD

**Issue by buildbreakdo** [nkzw-tech/athena-crisis#50]
Mouse input controls stop working, do not see the cursor on hover either. Looked in console and saw this:

<img width="554" alt="image" src="https://github.com/user-attachments/assets/a38d825a-a5a0-4d18-9a67-475d82882414">

On initial page load, error message is not present while the game is working. Have not figured out steps to reproduce. Happened once while scrolling to the bottom of the page, another time when coming back to a tab after a long period of time after a few moves. 

If I was to hazard a guess, mime type for an asset on an S3 bucket that does not exist is text/html 404 not found generic error page. Would be curious if assets are lazy loaded and when a new version is deployed if the old built files are being deleted. Thought is if that occurred while someone was playing a long game and a new version was deployed, old files deleted, then the game would end up at the state of requesting a JS asset that is no longer available and receive a 404 not found generic S3 error page of mime type text/html. Potential solution is never delete the old files. Just a wild hair of a guess.. 😅

Amazing project by the way, inspired. Cheers.

**Comment by cpojer** [comment]
Yep, that's unfortunately how it works with Cloudflare Pages at the moment. This is not a problem in the game itself since we cache all resources ahead of time for offline play.

I left a comment here about how I "solved" it for the game version before it had offline support: https://github.com/vitejs/vite/issues/11804#issuecomment-1833309687 – It's not ideal though.

For now, for the splash page, it's a wontfix.
